frontend/src/__tests__/cypress/cypress/pages/notebookServer.ts

```diff
@@ -0,0 +1,41 @@
+class NotebookServer {
+  visit() {
+    cy.visit('/notebookController/spawner');
+    this.wait();
+  }
+
+  private wait() {
+    cy.findByTestId('app-page-title');
+    cy.testA11y();
+  }
+
+  findAppTitle() {
+    return cy.findByTestId('app-page-title');
+  }
```

Comment:
Instead of writing it twice
```suggestion
  private wait() {
    this.findAppTitle()
    cy.testA11y();
  }

  findAppTitle() {
    return cy.findByTestId('app-page-title');
  }
```